.github/workflows/nemesis.yml

```diff
@@ -0,0 +1,76 @@
+name: Rust
+
+on:
+  push:
```

Comment:
we probably want to be more specific on what events this is triggered. Do we also want that to be a mandated workflow for merging? maybe we should

Reply:
good catch. WDYT doing same as Rust Tests: 

```
  push:
    branches: [ "main" ]
  pull_request:
  merge_group:
    branches: [ "main" ]
```

Reply:
>  Do we also want that to be a mandated workflow for merging? maybe we should

We should.

Reply:
sounds good to me :)

Reply:
I have updated the PR 🫡